src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,116 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private User user;
+    private RegistrationServiceImpl registrationService;
+
+    @AfterEach
+    void tearDown() {
+        registrationService.clearStorage();
+    }
+
+    @BeforeEach
+    void setUp() {
+        user = new User();
+        user.setId(1L);
+        user.setLogin("loginOne");
+        user.setAge(20);
+        user.setPassword("werewjdskfj");
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @Test
+    void userValidParameters_Ok() {
```

Comment:
```suggestion
    void userValidParameters_ok() {
```
кожна частина назви тестового методу повинна починатися з маленької літери, перевір це всюди